scripts/new_project.kts

```diff
@@ -1,25 +1,68 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("central", "https://repo.maven.apache.org/maven2/")
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.1")
+
+import kotlinx.coroutines.*
+import java.io.File
+import kotlin.coroutines.coroutineContext
+
 object NewProject {
 
     private const val KEY_APP_NAME = "app-name"
     private const val KEY_PACKAGE_NAME = "package-name"
     private const val ARGUMENT_DELIMITER = "="
+    private const val COROUTINE_FOLDER_NAME = "CoroutineTemplate"
+    private const val COROUTINE_PACKAGE_THIRD_SEG = "coroutine"
 
     private var appName = ""
     private var packageName = ""
+    private var appNameWithoutSpace = ""
 
-    fun generate(args: Array<String>) {
+    fun generate(args: Array<String>) = runBlocking {
         handleArguments(args)
+
+        showMessage("=> 🐢 Initializing new project...")
+        copyFiles(to = appNameWithoutSpace)
+        /*
+         * Set gradlew file as executable
+         * Copying file from one folder to another lost it's previous execution right.
+         * Need to set gradlew as executable to run
+         */
+        File("$appNameWithoutSpace/gradlew")?.setExecutable(true)
```

Comment:
Shall we put this into a new function, for example: `initializeProjectFolder()` / `initializeProjectDirectory()` ? 🙏